packages/eslint-config-eslint/index.js

```diff
@@ -13,17 +14,28 @@ const unicorn = require("eslint-plugin-unicorn");
 jsdoc.configs.recommended.plugins = { jsdoc };
 eslintComments.configs.recommended.plugins = { "eslint-comments": eslintComments };
 
-// extends eslint-plugin-n's recommended config
-const nodeConfigs = [nodeRecommendedConfig, {
+// extends eslint-plugin-n's config
+const nodeSharedRules = {
+    "n/callback-return": ["error", ["cb", "callback", "next"]],
+    "n/handle-callback-err": ["error", "err"]
+};
+const nodeCommonJSConfig = {
+    ...nodeRecommendedScriptConfig,
     rules: {
-        "n/callback-return": ["error", ["cb", "callback", "next"]],
-        "n/handle-callback-err": ["error", "err"],
-        "n/no-deprecated-api": "error",
+        ...nodeRecommendedScriptConfig.rules,
+        ...nodeSharedRules,
         "n/no-mixed-requires": "error",
         "n/no-new-require": "error",
         "n/no-path-concat": "error"
```

Comment:
These 3 rules will be in the `commonjs` config only because I believe they don't apply to ESM. Is that correct?